packages/node/src/lib/events/events.interface.ts

```diff
@@ -19,6 +23,21 @@ export interface ITriggerPayloadOptions extends IBroadcastPayloadOptions {
   to: TriggerRecipientsSubscribers;
   actor?: TriggerRecipientsSubscriber;
 }
+////////
+
+/**
+ * NEW INTEGRATION TYPES
+ * TODO
```

Comment:
Any todo's left here? Or will handle in another PR?

Reply:
In the next PR I will rearrange this. My eyes were cloudy at the time of finishing the PR.